frontend/src/pages/projects/screens/detail/notebooks/NotebookImageDisplayName.tsx

```diff
@@ -68,67 +82,49 @@ export const NotebookImageDisplayName = ({
         footer?: React.ReactNode;
       } => {
     if (notebookImage.imageStatus === NotebookImageStatus.DELETED) {
-      const unknownBody = (
-        <p>
-          An <b>unknown</b> notebook image has been deleted. To run this workbench, select a new
-          notebook image.
-        </p>
-      );
-      const knownBody = (
-        <p>
-          The <b>{notebookImage.imageDisplayName}</b> notebook image has been deleted. To run this
-          workbench, select a new notebook image.
-        </p>
-      );
       return {
         title: 'Notebook image deleted',
-        body: notebookImage.imageDisplayName ? knownBody : unknownBody,
+        body: (
+          <p>
+            This image version has been deleted. To continue using this workbench, select an
+            available version or a different workbench image.
+          </p>
+        ),
         variant: 'danger',
+        footer: updateToLatestButton,
       };
     }
     if (notebookImage.imageAvailability === NotebookImageAvailability.DISABLED) {
       return {
         title: 'Notebook image disabled',
         body: (
           <p>
-            The <b>{notebookImage.imageDisplayName}</b> notebook image has been disabled. This
-            workbench can continue using the image, but disabled images are not available for
-            selection when creating new workbenches.
+            This image version is disabled. This workbench can continue using this image version,
```

Comment:
```suggestion
            This image version is disabled. This workbench can continue using this version,
```
The disabled popover v2 removed this word